opennms-base-assembly/src/main/filtered/etc/opennms.properties

```diff
@@ -444,14 +444,6 @@ smslib.serial.polling=true
 # setting is true (See org.opennms.netmgt.provision.service.ProvisionService)
 #org.opennms.provisiond.enableDiscovery=true
 
-# Prior to 1.10 it was possible to delete entities that have been provisioned as
-# part of a provisioning group.  In 1.10 we have disabled this so that in order to
-# delete these entities you have to go back to the provisioning group and delete them
-# from there.  To reenable this deletion you can set this to true.
-# NOTE: if you do this then the object will be recreated when the provisioning group is
-# next imported/synchronized
-org.opennms.provisiond.enableDeletionOfRequisitionedEntities=true
```

Comment:
What would happen on those users that have the "old" default ? (i.e. enableDeletionOfRequisitionedEntities=false).